apps/server/src/functions/trpc/handler.ts

```diff
@@ -16,11 +43,46 @@ const createContext = (
     connectionString: process.env.DATABASE_URL,
   });
 
-// type Context = Awaited<ReturnType<typeof createContext>>;
+export const handler = async (
+  event: APIGatewayProxyEventV2
+): Promise<APIGatewayProxyResultV2> => {
+  // console.log("Full event:", JSON.stringify(event, null, 2));
+  
+  // Try multiple path sources
+  const path = 
+    event.rawPath || 
+    event.requestContext?.http?.path || 
+    event.pathParameters?.proxy ||
+    "";
+  
+  // Also check the body for tRPC batch requests
+  // const body = event.body ? JSON.parse(event.body) : null;
+  
+  // console.log("Extracted path:", path);
+  // console.log("Body:", body);
+  // console.log("Headers:", event.headers);
+  
+  // Block requests to /api/auth/* or /auth/*
+  if (path.includes("/api/auth") || path.includes("/auth")) {
+    console.log("Blocked auth route:", path);
+    return {
+      statusCode: 404,
+      body: JSON.stringify({ 
+        error: "Not found - auth routes are handled separately" 
+      }),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    };
+  }
 
-export const handler = awsLambdaRequestHandler({
-  router: appRouter,
-  createContext,
-});
+  // Pass through to tRPC handler
+  const trpcHandler = awsLambdaRequestHandler({
+    router: appRouter,
+    createContext,
+  });
+  
+  return trpcHandler(event);
```

Comment:
```suggestion
  return trpcHandler(event, {} as any);
```
Quick fix to get rid of the warning about the context argument missing.